src/box/relay.cc

```diff
@@ -468,6 +451,7 @@ relay_initial_join(struct iostream *io, uint64_t sync, struct vclock *vclock,
 	/* Freeze a read view in engines. */
 	struct engine_join_ctx ctx;
 	engine_prepare_join_xc(&ctx);
+	say_info("readview done");
```

Comment:
Looks like a debug print.

Reply:
This print is used in `replication_luatest/shutdown_test.lua`. But we decided it should be written another way and this version of test is just dropped. So the log is dropped too.